src/librustdoc/lint.rs

```diff
@@ -148,14 +148,13 @@ declare_rustdoc_lint! {
 }
 
 declare_rustdoc_lint! {
-    /// The `non_autolinks` lint detects when a URL could be written using
-    /// only angle brackets. This is a `rustdoc` only lint, see the
-    /// documentation in the [rustdoc book].
+    /// The `non_autolinks` lint detects when a URL is not a hyperlink.
```

Comment:
```suggestion
    /// The `bare_urls` lint detects when a URL is not a hyperlink.
```

Or should it be `rustdoc::bare_urls` in this prose?

Reply:
All the other lints use the bare name; I think I'd rather have it consistent. Happy to change it in a separate PR if you think it's useful.

Also right now the doc-comments aren't used for anything; there's an issue open to stop duplicating the same text in librustdoc and `src/doc/rustdoc`: https://github.com/rust-lang/rust/issues/81305